### PR TITLE
main CLI: add logging control args + turn on traceback for exceptions

### PR DESCRIPTION
**Pull request by yunjunz**
**Description of proposed changes**

+ add `mintpy --loglevel / quiet / verbose / debug` options to control the logging level, as suggested by @avalentino in #1017.

+ always turn ON traceback in case of exceptions, as suggested by @scottstanie in #1017

+ add example CLI usage

#168 

**Reminders**

- [x] Pass Pre-commit check (green)
- [x] Pass Codacy code review (green)
- [x] Pass [Circle CI test](https://app.circleci.com/jobs/github/yunjunz/MintPy/2178) (green)
- [x] Make sure that your code follows our style. Use the other functions/files as a basis.
- [x] If modifying functionality, describe changes to function behavior and arguments in a comment below the function declaration.
- [x] If adding new functionality, add a detailed description to the documentation and/or an example.

### CODE REVIEW

File: src/mintpy/__main__.py [hunk 1]
```diff
@@ -622,13 +674,17 @@ def main(*argv):
...
+        log.debug("args: %s", args)
+
+        exit_code = args.func(sys.argv[2:])
     except Exception as exc:
         log.critical(
             "unexpected exception caught: {!r} {}".format(
                 type(exc).__name__, exc)
         )
-        log.debug("stacktrace:", exc_info=True)
+        log.exception("stacktrace:", exc_info=True)
```
Comment (avalentino) [inline src/mintpy/__main__.py:687]: This is fine.
Please note, anyway, that, according to the documentation of the logging module, the `exc_info=True` is not strictly necessary for the `exception` method.

**Review by avalentino — APPROVED** [review]

File: src/mintpy/__main__.py (hunk 1 above)
Comment (yunjunz) [reply src/mintpy/__main__.py]: Commented out the `exc_info` arg. Thanks.

**Review by scottstanie — APPROVED** [review]
👍

File: src/mintpy/__main__.py [hunk 2]
```diff
@@ -41,7 +41,17 @@
 
 PROG = __package__
 LOGFMT = '%(asctime)s %(levelname)-8s -- %(message)s'
+DEFAULT_LOGLEVEL = "WARNING"
```
[the remark this answers is not in the document]
Comment (yunjunz) [reply src/mintpy/__main__.py:44]: Yes, it definitely should be on the list. Please feel free to edit them on the wiki (https://github.com/insarlab/MintPy/wiki).

**Pull request merged**